tests/test_coordinator.py

```diff
@@ -67,9 +67,38 @@ async def test_coordinator_async_update_with_data(mocker, coordinator):
             2: {"element": 3, "excluded": True, "id": 3, "index": 2, "name": "Outdoor Sensor 2", "status": False},
         },
         9: {
-            0: {"element": 1, "excluded": False, "id": 1, "index": 0, "name": "S1 Living Room", "status": True},
```

Comment:
Just to be sure I understand the change, why are we changing a test related to sectors? if the change (PR) is related to outputs management, it should only change outputs. I want to be sure we're not making assumptions that are not long lasting.

Reply:
Got it! we're good then!